config/file-dependency-updater/index.js

```diff
@@ -31,6 +31,27 @@ function removeFile(targetFilePath) {
     })
 }
 
+function isOSX() {
+    return process.platform === 'darwin';
+}
+
+function getOptions() {
+    const options = {
+        ignored: /(^|[\/\\])\../,
+        alwaysStat: true,
+        ignoreInitial: true
+    };
+    if (!isOSX()) {
```

Comment:
What worries me that chokidar seems to require different settings and behaves differently on different OSs. Since we use it to implement watching in our node-fs, we probably also should enable `awaitWriteFinish` there for Linux.